docs/concepts.md

```diff
@@ -0,0 +1,94 @@
+<!--[metadata]>
++++
+title = "Machine concepts and help"
+description = "Understand concepts for Docker Machine, including drivers, base OS, IP addresses, environment variables"
+keywords = ["docker, machine, amazonec2, azure, digitalocean, google, openstack, rackspace, softlayer, virtualbox, vmwarefusion, vmwarevcloudair, vmwarevsphere, exoscale"]
+[menu.main]
+parent="workw_machine"
+weight=3
++++
+<![end-metadata]-->
+
+
+# Understand Machine concepts and get help
+
+Docker Machine allows you to provision Docker on virtual machines that reside
```

Comment:
But tbh, vast majority of people will be creating virtual machines. Perhaps the fact that it can provision existing servers could be a side note.